### ISSUE TITLE
delete repository

### ISSUE THREAD

**Issue by setop** [betaglop/covid-19-alternative-medecine-recipes#1]
This content has no scientific background, is dangerous and should be deleted.

**Comment by betaglop** [comment]
Indeed, this is not scientific at all. This is personal feedback, with personal recipes (and not dangerous, often doctors even say this is only placebos, like homeopathy), this is marked down on the front page.

Although your comment is really interesting and I'll let it opened... to make sure ppl coming over will be able to read it : THIS IS NOT SCIENTIFIC AND IT IS NOT ITS PURPOSE.
